doc/module-docs/cc_ubuntu_autoinstall/data.yaml

```
@@ -0,0 +1,18 @@
+cc_ubuntu_autoinstall:
+  description: |
+    The `autoinstall` key indicates a configuration for the Ubuntu installer.
```

We generally use double-ticks around hard-coded string values and/or cloud-config keys. Single ticks result in italics which are hard to read/see on online documentation and italics generally have some usability concerns for those with dyslexia.
```suggestion
    The ``autoinstall`` key indicates a configuration for the Ubuntu installer.
```